generators/entity-client/templates/angular/src/main/webapp/app/entities/entity-management-dialog.component.html.ejs

```diff
@@ -21,12 +21,10 @@ const keyPrefix = angularAppName + '.'+ entityTranslationKey + '.';
 _%>
 <form name="editForm" role="form" novalidate (ngSubmit)="save()" #editForm="ngForm">
 
-    <div class="modal-header">
-        <h4 class="modal-title" id="my<%= entityClass %>Label" jhiTranslate="<%= keyPrefix %>home.createOrEditLabel">Create or edit a <%= entityClassHumanized %></h4>
-        <button type="button" class="close" data-dismiss="modal" aria-hidden="true"
-                (click)="clear()">&times;</button>
+    <div>
```

Comment:
the classes and filenames needs to be changed to remove dialog from it

Reply:
What is your suggestion for new name ? 
entity-management.component.html
entity-management-detail.component.html
entity-management-delete-dialog.component.html
entity-management-dialog.component.html --> ?

Reply:
entity-management-update.component.html
entity-management-dialog.component.ts

Reply:
i changed to update but have conflict .how can resolve that?

Reply:
You pull from our master, merge and resolve locally, then commit and push.